clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -106,10 +103,9 @@ public void setup() {
         commitRequestManager = testBuilder.commitRequestManager.orElseThrow(IllegalStateException::new);
         offsetsRequestManager = testBuilder.offsetsRequestManager;
         coordinatorRequestManager = testBuilder.coordinatorRequestManager.orElseThrow(IllegalStateException::new);
-        heartbeatRequestManager = testBuilder.heartbeatRequestManager.orElseThrow(IllegalStateException::new);
-        memberhipsManager = testBuilder.membershipManager.orElseThrow(IllegalStateException::new);
+        HeartbeatRequestManager heartbeatRequestManager = testBuilder.heartbeatRequestManager.orElseThrow(IllegalStateException::new);
```

Comment:
The variables `heartbeatRequestManager` and `membershipManager` are unused. Are they used to test the existence of `heartbeatRequestManager` and `membershipManager`? If so, could we rewrite them by `assertTrue`? For example: `assertTrue(testBuilder.heartbeatRequestManager.isPresent());`

Reply:
I don't think they are used to test the existence of the managers here, I would say they were just left unused so we should remove them. Managers are retrieved in this way in many other tests (ex [HeartbeatRequestManagerTest](https://github.com/apache/kafka/blob/525b9b1d7682ae2a527ceca83fedca44b1cba11a/clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java#L119)), but only when needed.

Reply:
btw, removing all  those unused might help us remove the suppression ClassDataAbstractionCoupling, worth checking

Reply:
Right, these can be removed and as @lianetm pointed this allows getting rid of the suppression. Thanks